ScoutSuite/frontend/src/pages/404/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import './style.scss';
+
+const ErrorPage = () => {
+
+  return (
+    <>
+      <div className="page-404">
+        <h1>Oh, no! It&apos;s a 404 page :&apos;(</h1>
```

Comment:
you could put the text in a string with `{}` so it's easier to read because of the apostrophes.